to_do_list/src/StorageManager.js

```diff
@@ -0,0 +1,13 @@
+class StorageManager {
```

Comment:
This file is way out of place, shouldn't be right under `src`, find a better place to put it.